### PR TITLE
2018 10 12

### PR DESCRIPTION
**Pull request by bmrsny**
Rough Draft

### CODE REVIEW

File: _posts/2018-10-09-elixir-casting-call.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+...(2)> end)
+...(2)> end)
+:ok
+iex(3)> :sys.get_state(pid)
+1000000
+{% endhighlight %}
+
+On every run of the above code, we will end up with the expected result of 1 million. All your hard earned cash is accounted for!
+
+
```
Comment (bmrsny) [inline _posts/2018-10-09-elixir-casting-call.md]: extra new line

File: _posts/2018-10-09-elixir-casting-call.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+ Actors have 3 main operations.
+
+1. Create another Actor/Process
+2. Send Message
+3. Handle Message
+
+#### The Power of Message Passing
+
+Each Process has it's own private state, processes that want to modify the state of another process do so by sending messages. Messages are received in the process's mailbox and handled one at a time with FIFO (First In First Out). This means that we won't have any "Dirty Reads" updating the state of our concurrent computation. We can use this mechanism, handling one message at a time, to update state not only in the process, but also in a memory store or DB without having inaccuracies. No need to craft clever locks!
```
Comment (laurawhalin) [inline _posts/2018-10-09-elixir-casting-call.md]: "it's" should be "its". Change the first comma to a period.

File: _posts/2018-10-09-elixir-casting-call.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  end
+
+  def handle_cast({:debit, amount}, state) do
+    {:noreply, state - amount}
+  end
+end
+{% endhighlight %}
+
+Our Generic Server process, although quite contrived, demonstrates how the Actor Model tracks and updates state.
+Next we need to pass messages to our BankAccount concurrently. We will simulate by using Task's. "Tasks are processes meant to execute one particular action throughout their lifetime, often with little or no communication with other processes. The most common use case for tasks is to convert sequential code into concurrent code by computing a value asynchronously. Elixir Docs 1.7"
```
Comment (laurawhalin) [inline _posts/2018-10-09-elixir-casting-call.md]: "Task's" should be "Tasks"

File: _posts/2018-10-09-elixir-casting-call.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+:ok
+iex(3)> :sys.get_state(pid)
+1000000
+{% endhighlight %}
+
+On every run of the above code, we will end up with the expected result of 1 million. All your hard earned cash is accounted for!
+
+
+## That's a wrap folks.
+Actor Model has a lot of powerful features and benefits. This is just the tip of the iceberg, nextime  we will work through supervising our BankAccount, walkthrough process communication types, and distributing our BankAccount across multiple nodes.
```
Comment (laurawhalin) [inline _posts/2018-10-09-elixir-casting-call.md]: "nextime" should be "next time"

File: _posts/2018-10-09-elixir-casting-call.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+author_github: bmrsny
+date:   2018-10-09 06:00:00
+comments: true
+banner_image: ElixirCasting.jpg
+---
+
+### Elixir's Actor Model
+
+Concurrent computation, it's unescapable. Elixir is known as a highly available concurrent language, but what does this really mean? Elixir's concurrency model is built on the Actor Model, but what is it?
+
```
Comment (laurawhalin) [inline _posts/2018-10-09-elixir-casting-call.md]: Add `<!--more-->` after this paragraph so we get more of a preview in the index.

**Comment by bmrsny** [comment]
@laurawhalin I addressed feedback.

**Pull request merged**